PopupBridge/build.gradle

```diff
@@ -22,11 +22,11 @@ android {
 }
 
 dependencies {
-    compile 'com.braintreepayments:browser-switch:0.1.4'
+    api 'com.braintreepayments:browser-switch:0.1.4'
```

Comment:
Holly made a good point that this version should be bumped to the current (0.1.5), but that it might be ideal to release browser switch first, then bump this to release with the new version.